### PR TITLE
package rtcqs

### PR DESCRIPTION
**Pull request by cbix**
https://codeberg.org/rtcqs/rtcqs

I may create another package for the `python-pysimplegui` optdepend, however that's not really proaudio-related ;)

### CODE REVIEW

**Comment by SpotlightKid** [comment]
> I may create another package for the `python-pysimplegui` optdepend

I assume you have seen https://aur.archlinux.org/packages/python-pysimplegui

File: packages/rtcqs/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+build() {
+  cd $pkgname-$pkgver
+  python -m build --wheel --no-isolation
+}
+
+package() {
+  cd $pkgname-$pkgver
+  python -m installer --destdir="$pkgdir" dist/*.whl
+  # license
+  install -Dm644 LICENSE -t "$pkgdir"/usr/share/licenses/${pkgname}
```
Comment (SpotlightKid) [inline packages/rtcqs/PKGBUILD:27]: Tiny nitpick: `$pkgname`

**Review by SpotlightKid — CHANGES_REQUESTED** [review]
The package installs the file

    /usr/lib/python3.10/site-packages/rtcqs/rtcqs_qt_gui.py

But when I try to run it, I get:

```
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.10/site-packages/rtcqs/rtcqs_qt_gui.py", line 3, in <module>
    import PySimpleGUIQt as sg
ModuleNotFoundError: No module named 'PySimpleGUIQt'
```

Maybe it's worth an upstream ticket.

When I run `rtcqs_gui` it shows the following warning on the "Audio Group" tab:

> User chris is currently not in the audio group. Add yourself to the audio group with 'sudo usermod -a -G audio chris' and log in again. See also https://wiki.linuxaudio.org/wiki/system_configuration#audio_group

The correct group name to check on Arch Linux would be `realtime`.

Maybe also worth an upstream ticket.

**Review by SpotlightKid — COMMENTED** [review]
Does upstream plan a new release including the `.desktop` file soon? Otherwise it might be worth adding the file from the repo to the `source` array and installing it manually.

File: packages/rtcqs/PKGBUILD (hunk 1 above)
Comment (cbix) [reply packages/rtcqs/PKGBUILD]: f5f1a65e3ddce451167c8f7d7ed70dd56a7f6b94

**Comment by cbix** [comment]
The Qt GUI is considered beta and neither included as a console script under `/usr/bin` nor installed as executable. I'd say we should not expect users trying to run that. If you want I can just remove it during build.

**Comment by cbix** [comment]
> The correct group name to check on Arch Linux would be `realtime`.

Interesting, do you have a reference for this? For some reason I have both groups on my system.

**Comment by SpotlightKid** [comment]
* https://wiki.archlinux.org/title/Realtime_process_management
* https://archlinux.org/packages/community/any/realtime-privileges/

**Comment by SpotlightKid** [comment]
> If you want I can just remove it during build.

I think it's ok to just leave it there. It's a 0.x release after all.

**Review by SpotlightKid — APPROVED** [review]
LGTM

**Comment by cbix** [comment]
@SpotlightKid I opened this issue now https://codeberg.org/rtcqs/rtcqs/issues/4

**Pull request merged**